### PR TITLE
Updating Inventory service and controller with create microservice

### PR DESCRIPTION
**Pull request by valentinaduarte**

### CODE REVIEW

File: src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/InventaryService.java [hunk 1]
```diff
@@ -1,24 +1,51 @@
...
+    /*--------------------
+     * CREATE INVENTORY
+     * -------------------
+     */
+
+    @Override
     public InventaryResponse create(InventaryRequest request) {
-        // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'create'");
+        Date fechaActual = new Date();
```
Comment (juansefdz) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/InventaryService.java]: Date fechaActual = new Date();

Remember that variables must be declared in English and in camelCase format.

File: src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/InventaryController.java [hunk 2]
```diff
@@ -1,48 +1,61 @@
...
+    * -------------------
+    */
 
-     //insert here your code
+    // insert here your code
 
     /*--------------------
      * CREATE INVENTARY
      * -------------------
      */
```
Comment (juansefdz) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/InventaryController.java]: add @PostMapping

**Review by juansefdz — CHANGES_REQUESTED** [review]

File: src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/InventaryService.java (hunk 1 above)
Comment (valentinaduarte) [reply src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/InventaryService.java]: I've already made the updates

File: src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/InventaryController.java (hunk 2 above)
Comment (valentinaduarte) [reply src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/InventaryController.java]: I've already made the updates

**Review by juansefdz — APPROVED** [review]

**Pull request merged**